Task 2/script.js

```diff
@@ -0,0 +1,14 @@
+/* ------------------------------ TASK 2 ----------------------------
+Parašykite JS kodą, kuris skaičiuos kiek kartų buvo paspaustas mygtukas
+su tekstu "CLICK ME". Paspaudimų rezultatas turi būti matomas dešinėje
+pusėje esančiame "state" skaičiavimo bloke (<div id="btn__state">0</div>)
+------------------------------------------------------------------- */
+
+const counter =  document.getElementById('btn__state');
+const buttonClick = document.getElementById('btn__element');
+
+function countUp() {
+    counter.innerHTML++;
```

Comment:
This solutions works, but i suggest to have separate variable that tracks counter inside of Javascript, then increase it on every click and update it in HTML.